<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteMachineScanDialog</class>
 <widget class="QDialog" name="RemoteMachineScanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scan remote machine</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="protocolComboBox"/>
   </item>
   <item>
    <widget class="QTableWidget" name="machinesTableWidget">
     <column>
      <property name="text">
       <string>Add</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Host</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Protocol</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPushButton">
       <property name="text">
        <string>Add selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
